manifests/node/kubelet.pp

```diff
@@ -228,21 +236,35 @@
   Package <| title == 'containernetworking-plugins' |> -> Service['kubelet']
 
   if $manage_firewall {
-    firewalld_custom_service { 'kubelet':
-      ensure      => $ensure,
-      short       => 'kubelet',
-      description => 'Kubernetes kubelet daemon',
-      ports       => [
-        {
-          port     => '10250',
-          protocol => 'tcp',
-        },
-      ],
-    }
-    firewalld_service { 'Allow k8s kubelet access':
-      ensure  => $ensure,
-      zone    => 'public',
-      service => 'kubelet',
+    case fact('os.family') {
+      'Debian': {
+        include firewall
+
+        firewall { '100 allow kubelet access':
+          dport  => 10250,
+          proto  => 'tcp',
+          action => 'accept',
```

Comment:
Do we need to make this port available public? Or should this be limited to a specifc network/cidr?